packages/styled-system/src/tokens/useInjectTokens.tsx

```diff
@@ -0,0 +1,49 @@
+import { useInsertStyleElement } from "../utils/useInsertStyleElement.ts";
+import { Tokens } from "./tokens.ts";
+
+/**
+ * Injects the tokens into the DOM as CSS variables, under the rootSelector className.
+ * @example
+ * assuming different versions of the design system loaded at the same time, one with version 1, the other with version 2.0.0
+ *  <head>
+ *      <style id="hop-tokens-1.0.0">
+ *          .hop-1-0-0 {
+ *              --token: red
+ *          }
+ *       </style>
+ *       <style id="hop-tokens-2.0.0">
+ *          .hop-2-0-0 {
+ *              --token: blue
+ *          }
+ *       </style>
+ *   </head>
+ */
+export function useInjectTokens(rootSelector: string) {
+    useInsertStyleElement(
```

Comment:
We should probably bundle all these? or keep the dark separate just like our tokens?

Reply:
I'm not against many <style> I've seen it on many websites.